shared/src/commonMain/kotlin/com/mbta/tid/mbta_app/model/ObjectCollectionBuilder.kt

```diff
@@ -163,13 +163,13 @@ class ObjectCollectionBuilder {
     class RouteBuilder : ObjectBuilder<Route> {
         var id = uuid()
         var type = RouteType.LIGHT_RAIL
-        var color = ""
+        var color = "FFFFFF"
```

Comment:
Got errors when writing RoutePill tests that the color code needed to be at least 6 characters - setting that by default on all test objects.